lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -1833,7 +1916,16 @@ class EngineSemanticsOwner {
       for (final SemanticsObject removal in removals) {
         _semanticsTree.remove(removal.id);
         removal._parent = null;
-        removal.element.remove();
+
+        // This is the trick used to prevent the screen reader from focusing on
+        // the <body> tag
+        if (removal.element != focusedElement) {
+          // Element is not focused: safe to remove from DOM.
+          removal.element.remove();
+        } else {
+          // Element is focused: not safe to remove from DOM. Leave a tombstone.
+          removal.leaveTombstone();
```

Comment:
`leaveTombstone` doesn't describe very well what the method does. What about `.removeOnBlur()`?